doc/2/api/controllers/security/create-first-admin/index.md

```diff
@@ -68,7 +69,7 @@ Body:
 
 - `_id`: specify the administror [kuid](/core/2/guides/main-concepts/authentication#kuzzle-user-identifier-kuid), instead of letting Kuzzle generate a random identifier.
 - `reset` (boolean): if true, restricted rights are applied to the `anonymous` and `default` roles (by default, these roles don't have any restriction).
-
+- `kuid`: if set to `human`, Kuzzle will generate a human readable id, otherwise if set to `uuid` Kuzzle will generate a standard uuid (default: `"human"`)
```

Comment:
Actually, it would be cool that the auto-generated KUID for the first admin contain the word `admin` or something similar.

Reply:
Since kuid cannot be easily changed, what if admin priviledges get removed from that user?

Reply:
I agree with @scottinet . Moreover in the Admin Console, it's easy to differentiate an admin via his profiles so it's not embarrassing IMO
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/7868838/106780169-04d9e780-6648-11eb-9a00-2194004ccfe6.png">